src/style.rs

```diff
@@ -313,18 +313,51 @@ impl ProgressStyle {
                                     HumanDuration(state.started.elapsed())
                                 ))
                                 .unwrap(),
-                            "per_sec" => buf
-                                .write_fmt(format_args!("{:.4}/s", state.per_sec()))
-                                .unwrap(),
-                            "bytes_per_sec" => buf
-                                .write_fmt(format_args!("{}/s", HumanBytes(state.per_sec() as u64)))
-                                .unwrap(),
-                            "binary_bytes_per_sec" => buf
-                                .write_fmt(format_args!(
-                                    "{}/s",
-                                    BinaryBytes(state.per_sec() as u64)
-                                ))
-                                .unwrap(),
+                            "per_sec" => {
+                                if matches!(&state.status, Status::InProgress) {
+                                    buf.write_fmt(format_args!("{:.4}/s", state.per_sec()))
```

Comment:
I'm open to this change, but this way seems unnecessarily repetitive. It looks like we can put the logic here into a separate `ProgressState` method?